src/components/graph/Graph.jsx

```diff
@@ -400,7 +414,7 @@ export default class Graph extends React.Component {
 
         return (
             <div id={`${this.state.id}-${CONST.GRAPH_WRAPPER_ID}`}>
-                <svg style={svgStyle}>
+                <svg name="canvas" style={svgStyle} onClick={this.onClickCanvas}>
```

Comment:
Again regarding the canvas reference, please rename it to `svg-container-${this.state.id}`. Why appending the `this.state.id`? To avoid collisions when there are more than one graph present in the same page (yes, this is a real use case)